bench/Chainweb/Pact/Backend/Bench.hs

```diff
@@ -45,7 +45,7 @@ import Chainweb.Utils.Bench
 import Chainweb.Utils (sshow)
 import Chainweb.Version
 
-v :: ChainwebVersion
+v :: ChainwebVersion dc
```

Comment:
What do the letters `dc` mean?

Reply:
Devnet config.

Reply:
I don't really like this name, how bout `c`? or `cfg`?